src/nft/components/profile/list/Modal/SuccessScreen.tsx

```diff
@@ -0,0 +1,128 @@
+import { Trans } from '@lingui/macro'
+import Column from 'components/Column'
+import Row from 'components/Row'
+import useStablecoinPrice from 'hooks/useStablecoinPrice'
+import useNativeCurrency from 'lib/hooks/useNativeCurrency'
+import { getTotalEthValue } from 'nft/components/bag/profile/utils'
+import { useSellAsset } from 'nft/hooks'
+import { formatEth, generateTweetForList, pluralize } from 'nft/utils'
+import { useMemo } from 'react'
+import { Twitter, X } from 'react-feather'
+import styled, { css, useTheme } from 'styled-components/macro'
+import { BREAKPOINTS, ThemedText } from 'theme'
+import { formatDollar, priceToPreciseFloat } from 'utils/formatNumbers'
+
+import { TitleRow } from '../shared'
+
+const SuccessImage = styled.img<{ numImages: number }>`
+  width: calc(${({ numImages }) => (numImages > 1 ? (numImages > 2 ? '33%' : '50%') : '100%')} - 16px);
```

Comment:
what does the `- 16px` do here and can it not be changed to padding instead?

Reply:
It accounts for the gap from the spacing between images. Attempting to use padding instead causes issues with the border radius